README.md

```diff
@@ -50,9 +50,13 @@ assets.
 ```shell
 build/service/opendigitizer-service &         # launches the service
 build/ui/opendigitizer_ui &                   # launches the native digitizer UI
-xdg-open http://localhost:8080/web/index.html # launches the webassembly UI
-xdg-open http://localhost:8080/flowchart      # launches the html based web ui for the flowgraph property
-xdg-open http://localhost:8080/acquisition    # launches the html based web ui for the acquisition property
+export CHECK_CERTIFICATES=1                   # to disable certificate checking
```

Comment:
Is the logic really `CHECK_CERTIFICATES=1` -> disable checking? IMO either the comment is wrong or the env variable naming/semantic is misleading.

Reply:
oh yes, that comment is wrong

Reply:
it was the comment